### ISSUE TITLE
Original memory requests/limits values are incorrect on the grafana dashboard

### ISSUE THREAD

**Issue by chandrams** [kruize/kruize#151]
**Describe the bug**
Original memory requests/limits values are incorrect on the grafana dashboard

**To Reproduce**
Steps to reproduce:
- Deploy kruize using the below command:
./deploy.sh -ac docker -i chandra25ms/kruize:0.6.6 

- Deploy acmeair or any application by specifying cpu/memory requests
 memory: 500M
 cpu: 2
 limits:
 memory: 1024M
 cpu: 3

I have attached the screenshot, the values for acmeair shown are 500B and 1kB instead of 500M and 1024M

**Expected behavior**

**Screenshots**

**Environment (please complete the following information):**
 - OS: Ubuntu
 - Kubernetes Environment Minikube
**Additional context**

**Comment by chandrams** [comment]
![Screenshot_2020-07-14 Kruize Dashboard - Grafana](https://user-images.githubusercontent.com/9799411/87431613-84e52500-c604-11ea-8d12-6185e4b78c0f.png)

**Comment by chandrams** [comment]
From acmeair.yaml

```
        resources:
          requests:
            memory: 500M
            cpu: 2
          limits:
            memory: 1024M
            cpu: 3
```

**Comment by chandrams** [comment]
Verified the fix in PR #152 on minikube, both original cpu and mem req/limits are displayed. With Docker, original CPU limits are not displayed in the grafana dashboard for --cpus and --cpu-period, but displayed with --cpu-quota.

Tested with the below CPU values:
```
docker run --rm -d --name=acmeair-mono-app1 --cpus="2.5" --memory="1024M"  -p 32333:8080 --network=acmeair-net -e MONGO_HOST='acmeair-db1' acmeair_mono_service_liberty1

docker run --rm -d --name=acmeair-mono-app2 --cpus="2" --memory="2048M"  -p 32334:8080 --network=acmeair-net -e MONGO_HOST='acmeair-db1' acmeair_mono_service_liberty1

docker run --rm -d --name=acmeair-mono-app3 --cpu-period="100000" --cpu-quota="150000" --memory="2048M"  -p 32335:8080 --network=acmeair-net -e MONGO_HOST='acmeair-db1' acmeair_mono_service_liberty1

docker run --rm -d --name=acmeair-mono-app4 --cpu-period="200000" --cpu-quota="150000" --memory="4096M"  -p 32336:8080 --network=acmeair-net -e MONGO_HOST='acmeair-db1' acmeair_mono_service_liberty1

docker run --rm -d --name=acmeair-mono-app5 --cpu-quota="200000" --memory="4096M"  -p 32337:8080 --network=acmeair-net -e MONGO_HOST='acmeair-db1' acmeair_mono_service_liberty1
```

Contents of kruize-docker.json from the kruize container for the above acmeair app instances:
```
{
  "containers": [
    { "name": "cadvisor", "cpu_limit": "0", "mem_limit": "0" },
    { "name": "grafana", "cpu_limit": "0", "mem_limit": "0" },
    { "name": "kruize", "cpu_limit": "0", "mem_limit": "0" },
    { "name": "prometheus", "cpu_limit": "0", "mem_limit": "0" },
    { "name": "acmeair-mono-app1", "cpu_limit": "0", "mem_limit": "1073741824" },
    { "name": "acmeair-mono-app2", "cpu_limit": "0", "mem_limit": "2147483648" },
    { "name": "acmeair-mono-app3", "cpu_limit": "1", "mem_limit": "2147483648" },
    { "name": "acmeair-mono-app4", "cpu_limit": "0", "mem_limit": "4294967296" },
    { "name": "acmeair-mono-app5", "cpu_limit": "2", "mem_limit": "4294967296" },
    { "name": "petclinic-app", "cpu_limit": "0", "mem_limit": "0" }
  ]
}
```
